### DISCUSSION TITLE
What is the default value for `biome ci` `--formatter-enabled`, `--linter-enabled`, and `--organize-imports-enabled`?

### DISCUSSION THREAD

**Discussion by yamcodes** [biomejs/biome#487] [Q&A] [answered]
Can we specify the default values for the _available options_ section of the [`biome ci` docs](https://biomejs.dev/reference/cli/#biome-ci)?

Additionally, it would be great to add more information regarding these default values in the [Continuous Integration recipe](https://biomejs.dev/recipes/continuous-integration/).

I can open a PR once provided this information for all three parameters.

Thank you.

**Comment by ematipico** [comment, marked as answer]
You could say that their default values are `undefined` because they are **optional** options. These options are meant to **override** the values coming from Biome's default and/or `biome.json`. 

This means that if these options **aren't** passed via CLI, Biome will send the values coming from its defaults/config file. If these options are passed, they will override the values accordingly. 

> I can open a PR once provided this information for all three parameters.

The CLI reference page is auto-generated from the source code, so you would want to update the source comments: https://github.com/biomejs/biome/blob/main/crates/biome_cli/src/commands/mod.rs#L68
